src/internal/FluencePeer.ts

```diff
@@ -102,6 +95,11 @@ export interface PeerConfig {
      * If the option is not set default TTL will be 7000
      */
     defaultTtlMs?: number;
+
+    /**
+     * Plugable AVM runner implementation. If no specified a single-thread, ui-blocking runner will be used.
```

Comment:
```suggestion
     * Plugable AVM runner implementation. If no specified a single-thread, ui-blocking runner will be used.
```

Is this still true? We're using single-thread by default?